get-started/first-steps-vs.md

```diff
@@ -21,29 +21,41 @@ In this tutorial, you will enhance an existing .NET MAUI application by adding a
 
-1. Open Visual Studio and select **Create a new project** in the start window.
+* <a href="https://marketplace.visualstudio.com/items?itemName=TelerikInc.ProgressTelerikMAUIExtensions" target="_blank">The Visual Studio Marketplace.</a>
 
-1. Select the **.NET MAUI App** template, and click the **Next** button.
+## Step 2: Start the Telerik MAUI Extension
 
-	![Telerik UI for .NET MAUI - create new MAUI project in Visual Studio](./images/gs-vs-create-maui-app.png)
-	
-1. Name your project and select a location.
+To start the wizard, use the [Project](#project-menu) menu.
 
-1. Choose the .NET framework for your project.
+1. Open Visual Studio and choose the **Create a new project** option.
+1. Search for **telerik maui**:
 
-1. Wait until Visual Studio restores all dependencies (when done, all exclamation marks in the **Dependencies** tree view item disappear).
+    ![Create a new project dialog with maui in the search field and results](../installation/vs-integration/images/vsextensions_createapp.png)
 
-1. Click the **Windows Machine** button to build and run the app.
+1. Select one of the supported project templates:
 
-	![Telerik UI for .NET MAUI - create new MAUI project in Visual Studio](./images/gs-vs-build-run.png)
+     * **Telerik .NET MAUI App - Configurable Wizard**&mdash;Represents a configurable wizard, which contains various predefined application templates that include the Telerik .NET MAUI suite setup. 
+     * **Telerik .NET MAUI Blank App**&mdash;Represents a blank project for creating a .NET MAUI application that includes the Telerik .NET MAUI suite setup.
+     * **Telerik .NET MAUI Blank Shell App**&mdash;Creates a blank .NET MAUI Shell application. You can use this project as a .NET MAUI Shell example. In addition, you can use features like Shell Navigation, .NET MAUI Shell flyout, and .NET MAUI Shell tabs. For more details on .NET MAUI Shell, review the official [Microsoft documentation](https://learn.microsoft.com/en-us/dotnet/maui/fundamentals/shell/). 
 
-If you encounter any issues creating the basic project, see the complete guide in <a href="https://learn.microsoft.com/en-us/dotnet/maui/get-started/first-app?pivots=devices-windows&view=net-maui-8.0&tabs=vswin" target="_blank">Microsoft's .NET MAUI documentation</a>. 
 
-## Step 2: Add the Telerik NuGet Server
+## Step 3: Configure the Project
+
+1. The next step lets you configure your **Telerik UI for .NET MAUI** project by setting your project name and its location.
+
+    ![Telerik .NET MAUI App configurable wizard initial screen within the Create your new project dialog](../installation/vs-integration/images/vsextensions_configureapp.png)
+
+1. (Optional) When using the **Telerik .NET MAUI App - Configurable Wizard** project template, an additional screen for choosing the predefined application template loads:
```

Comment:
Consider that we must offer only one option as this is a straightforward tutorial. No optional steps should be offered.